### ISSUE TITLE
Make `untie_biases` accept tuples

### ISSUE THREAD

**Issue by f0k** [Lasagne/Lasagne#249]
The convolutional layers have a keyword argument `untie_biases`, which is a boolean. If set to `False`, the layer will learn a bias value per input channel. If set to `True`, it will learn a bias value per input channel, height and width. In concordance with `filter_size` and `stride`, it should also accept a tuple of boolean denoting whether to untie the biases for each dimension of a 2D or 3D convolution. One use case is to learn separate biases for the frequency components, but shared biases for the time components of a spectrogram.

**Comment by benanne** [comment]
I don't think this is really "in concordance with `filter_size` and `stride`" but I guess it could be a useful feature to have. Ideally it would have to be implemented for all 4 convolution implementations though.

**Comment by f0k** [comment]
> I don't think this is really "in concordance with `filter_size` and `stride`"

Well, both of them allow to either specify a separate value per spatial dimension or a single value that's used for all spatial dimensions at once. Could be exactly the same for `untie_biases`, without breaking compatibility to existing code.

> Ideally it would have to be implemented for all 4 convolution implementations though.

Sure. We might want to try getting rid of some of the code duplication in there... That's low priority, though.

**Comment by benanne** [comment]
agreed. Well, no objection from me at any rate. I guess we can reuse `as_tuple`. It may get a little tricky in combination with the `c01b` ordering in `Conv2DCCLayer` when `dimshuffle=False`, but it should be doable nevertheless.

**Comment by f0k** [comment]
From the discussion in https://github.com/Lasagne/Lasagne/pull/247#discussion_r30406024, maybe another alternative would be to have `untie_biases` accept a list of axes that defaults to `(1,)` (or `(0,)` for `c01b=True`) if not given? That would break the symmetry to `filter_size` and `stride`, but allow controlling parameter tying for the channels as well, and possibly introduce a symmetry to `untie_alphas` in `PReLULayer`.
